Sources/UIView.swift

```diff
@@ -258,7 +258,7 @@ open class UIView: UIResponder, CALayerDelegate, UIAccessibilityIdentification {
         let otherAbsoluteOrigin = otherView.absoluteOrigin()
 
         let originDifference = (otherAbsoluteOrigin - selfAbsoluteOrigin)
-        return point - originDifference
+        return (point - originDifference).applying(otherView.superview?.transform.inverted() ?? .identity)
```

Comment:
This code was moved from `Sources/UIPanGestureRecognizer.swift`. It's still a hack, in my current understanding we have to consider all transforms between self and otherView (which we partially do in `absoluteOrigin` 🤯 )

Reply:
It would be great if you could add a code comment that this is still kind of a hack and ideally it would incorporate all transforms up the tree